Cargo.toml

```diff
@@ -1,31 +1,33 @@
 [package]
-name = "fuse"
-version = "0.3.0"
 authors = ["Andreas Neuhaus <zargony@zargony.com>", "Eric Delanghe <delanghe.eric.dev@gmail.com>"]
+build = "build.rs"
+categories = ["api-bindings", "filesystem"]
 description = "Rust library for filesystems in userspace (FUSE)"
 documentation = "https://docs.rs/fuse"
 homepage = "https://github.com/zargony/rust-fuse"
-repository = "https://github.com/zargony/rust-fuse"
-readme = "README.md"
 keywords = ["fuse", "filesystem", "system", "bindings"]
-categories = ["api-bindings", "filesystem"]
 license = "MIT"
-build = "build.rs"
+name = "fuse"
+readme = "README.md"
+repository = "https://github.com/zargony/rust-fuse"
+version = "0.3.0"
 
 [dependencies]
+cfg-if = "0.1.2"
 errno = "0.2.3"
 libc = "0.2"
 log = "0.3"
-time = "0.1"
-thread-scoped = "1"
 sendfd = "0.1.0"
+thread-scoped = "1"
+time = "0.1"
+getopts = "0.2.15"
```

Comment:
Optional ? Only needed for rust-mount. http://doc.crates.io/manifest.html#the-features-section

Reply:
done 7910f37